xml/art_sle_modules_quick.xml

```diff
@@ -1152,8 +1152,16 @@ Package  Module or Repository
 emacs    Basesystem Module (sle-module-basesystem/15/x86_64)
 emacs    Available</screen>
   </example>
- </sect1>
 
+  <para>
+   If you do not want to search for a certain package, but want to view the list
```

Comment:
How about: "Instead of searching for packages, you can also view the list..."?

Reply:
Much better, thanks.